### ISSUE TITLE
feat: Display Reference Name instead of actual value

### ISSUE THREAD

**Issue by fbakhda** [sabrogden/Ditto#577]
Most of us today do remote work and a lot of the time, we have video calls and at times we have video calls that are being recorded. 
This can lead to sensitive information from your clipboard while you are pasting be available for the viewer to pause and look at it. 

Hence there is possibility that sensitive copy paste information can be leaked out. 

I am proposing to have an additional feature, where after a copy being added to Ditto, the user can "label" the copy and show the label name instead of the actual value on the quick paste window. This would help to hide the sensitive information from being leaked.

**Comment by PzaThief** [comment]
You can title the clip in `Quick list` -> `Select clip` -> `Properties` (Alt+Enter) -> Item Title.
The title is just for label as you want, it doesn't make any change of copied content.

**Comment by fbakhda** [comment]
OMG! This is so awesome! I had no idea on this. I really thought the Title is the copied item. 
Perhaps and interactive rename function on the quick list itself would help this feature to be more prominent!

But thank you, this resolves my issue.

**Comment by PzaThief** [comment]
Indeed. Like any other Ditto hidden features, this is a hidden action that is difficult to use.
but I think Ditto seems to have so few contributor compared to users to apply a lot of good ideas like yours.
I hope more people come to here so Ditto will be boosted.
btw, happy new year~

**Comment by fbakhda** [comment]
I would love to help, but CPP and/or C itself is horrible. 
If you do have like short videos or trainings on your codebase then perhaps as a developer buddy, simple ideas could be easily implemented, could possibly do and send you a pull request :)

~~ HAPPY NEW YEAR ~~

**Comment by Chealer** [comment]
@fbakhda wrote:
> I had no idea on this. I really thought the Title is the copied item. Perhaps and interactive rename function on the quick list itself would help this feature to be more prominent!

It can't simply offer to "rename", since:
1. It is a title, not a name
2. Most clips have no title.

But I agree this can easily be improved. Ticket #698, filed a few months after your request, suggests you are not alone.

This is an ITS. Your ticket seems related to ticket #635. If it does not track an otherwise untracked issue, please discard it. If it does, its title should first focus on describing the issue, not just suggest solutions.

@PzaThief wrote:
> but I think Ditto seems to have so few contributor compared to users to apply a lot of good ideas like yours.
> I hope more people come to here so Ditto will be boosted.

Amen
